cmd/cloud.go

```diff
@@ -164,20 +155,26 @@ This will execute the test on the k6 cloud service. Use "k6 login cloud" to auth
 				}
 			}
 
-			if err = cloudapi.MergeFromExternal(arc.Options.External, &cloudConfig); err != nil {
+			// Cloud config
+			cloudConfig, err := cloudapi.GetConsolidatedConfig(
+				derivedConf.Collectors["cloud"], osEnvironment, "", arc.Options.External)
+			if err != nil {
 				return err
 			}
+			if !cloudConfig.Token.Valid {
+				return errors.New("Not logged in, please use `k6 login cloud`.") //nolint:golint,revive,stylecheck
```

Comment:
Is expected to have a capital letter?

Reply:
It's a message that gets shown to the user, so it should probably be a full sentence. 

Also, this is a problem across the codebase so if you want to change that, please open a separate issue ;)